README.md

```diff
@@ -1,8 +1,8 @@
-<p align="center">
-  <a href="https://emberjs.com"><img width="300" src="https://raw.githubusercontent.com/emberjs/website/master/source/images/brand/ember_Ember-Light.png"></a>
+<p style="text-align:center">
```

Comment:
GitHub strips `style` attributes in their rendered markdown: this change causes the logo and badges to be left-aligned in the rendered output rather than centered.

README on `master`:
<img width="887" alt="the top of the `emberjs/ember.js` readme with the logo and badges centered" src="https://github.com/emberjs/ember.js/assets/108688/519dd198-66d5-420f-be80-0b050f47668d">

README on this branch:
<img width="886" alt="the top of the `emberjs/ember.js` readme with the logo and badges left-aligned" src="https://github.com/emberjs/ember.js/assets/108688/da004890-63c1-4895-afb5-57efe49e940d">

Reply:
@dfreeman Thanks for pointing it out. The change is not as desired in the markdown per github strip style.